shared/gh/partials/admin-batch-edit.html

```diff
@@ -32,8 +32,20 @@
                     </li>
                 </ul>
 
+                <!-- Series title -->
                 <% if (data.borrowedFrom) { %><i class="fa fa-link pull-left"></i> <% } %><h1 class="gh-jeditable-series-title"><%- data.series.displayName %></h1>
 
+                <!-- Borrowed data -->
+                <% if (data.borrowedFrom) { %>
```

Comment:
`<% if (data.borrowedFrom && (data.tripos || data.part)) { %>` will be more solid I think